docs/DeveloperGuide.md

```diff
@@ -505,6 +505,27 @@ Priorities: MVP (must have), 2 (nice to have), 3 (unlikely to have)
       Steps 1a1 to 1a2 are repeated until the data entered are correct.  
       Use case resumes from Step 2.
 
+**Use Case: UC08 - Find lessons with a specific name keyword**
+
+**MSS**:
+
+1. Tutor keys in a keyword in the required field to find lessons whose names contain this keyword.
+1. TutorEase lists the lessons with the given keyword.  
+   Use case ends.
+
+**Extensions**:
+
+* **1a**. TutorEase detects bad or wrongly formatted inputs.
```

Comment:
Should we give more details about different 'bad or wrongly formatted inputs' for different commands? etc. number , special characters?

Reply:
I thought that use case do not include a lot of implementation details? Let's see what the rest think